allegro.go

```diff
@@ -98,7 +100,7 @@ func handleRequests() {
     router := httprouter.New()
     router.GET("/", Index)
     router.POST("/", VerifyPayloadAndSend)
-    log.Fatal(http.ListenAndServe(":3000", router))
+    log.Fatal(http.ListenAndServe(":8081", router))
```

Comment:
Maybe we should make all the ports environment variables. Would probably simplify things.